src/pages/index.js

```diff
@@ -95,10 +96,12 @@ export default function WRMCWebsite() {
     // TODO: we should create a container component for the Home page
     // (and any other pages that end up having multiple components)
     "Home" : <div>
-              <ShowOTW show={sotw} handleClick={clickShow}/> 
+              <ShowOTW show={sotw} handleClick={clickShow}/>
               <p>{""}</p>
               <DisplayCurrentShow show={isOnAir ? upcomingShows[0] : shows.find(show => show.id === 12345)} handleClick={clickShow}/>
               <p>{""}</p>
+              <DisplayCurrentPlaylist playlist = {currentPlaylist ? currentPlaylist : []} allSongs = {allSongs}/>
```

Comment:
This might cause a proptypes error, as `currentPlaylist` is an object, but sometimes we are passing in an empty list here.

Reply:
I did two things. First, I made it so that playlistType isn't required because an empty playlist may still sneak through if a show is started and a song isn't immediately entered. Second, I created an empty playlist object to be the false condition.